src/program/lwaftr/run_nohw/README.inc

```diff
@@ -1,8 +0,0 @@
-Usage: run-nohw --help
```

Comment:
Moving the text to the `README` file, and turning `README.inc` into a symbolic link, as it is for the other commands. Not sure how this works but hey, consistency. ;-)